version_check.go

```diff
@@ -12,12 +12,10 @@
 // implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-// +build !darwin !amd64
+// +build !go1.11
 
-package sigtramp
+package libedit
 
-import "unsafe"
-
-func Get() unsafe.Pointer {
-	return unsafe.Pointer(uintptr(0))
+func init() {
+  panic("go-libedit requires go1.11+")
```

Comment:
you mean go1.12 in the error message

Reply:
I don't think so? The signal handling changes landed in go1.11, unless I'm mistaken: https://github.com/golang/go/commit/ec989337c5d3203d52de1a2314813996c711fce6

Reply:
oops, yes, you're right. merge away :)